man/systemd.exec.xml

```diff
@@ -807,11 +807,51 @@ CapabilityBoundingSet=~CAP_B CAP_C</programlisting>
         names must be relative, and may not include <literal>..</literal>. If set, one or more
         directories by the specified names will be created (including their parents) below <filename>/run</filename>
         (or <varname>$XDG_RUNTIME_DIR</varname> for user services), <filename>/var/lib</filename> (or
-        <varname>$XDG_CONFIG_HOME</varname> for user services), <filename>/var/cache</filename> (or
+        <varname>$XDG_DATA_HOME</varname> for user services), <filename>/var/cache</filename> (or
```

Comment:
No, XDG_DATA_HOME corresponds with /usr/share/, while we are referring to a writable state directory, i.e. something corresponding with /var/lib/. The XDG spec simply doesn't distuingish between state and configuration, and that's probably OK as the line is very blurry in desktop apps...

Reply:
Yes, XDG_DATA_HOME  correspond to /usr/share which is different from /var/lib.
I just tested and indeed *State* goes in `.config`. Better document what is true (and test before submit)
Ideally, having the "state" in something called "DATA" looked better, but, yeah, well, ...